serialization/src/main/java/com/twitter/serial/stream/bytebuffer/ByteBufferSerializerInput.java

```diff
@@ -262,42 +262,7 @@ private long readLongHeader(byte expectedType) throws IOException {
 
     @NotNull
     private String decodeUtf8String(int length) throws IOException {
```

Comment:
Since function is private you can just remove it without modifying the public API.